app/views/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+    <link href='https://fonts.googleapis.com/css?family=Bree+Serif' rel='stylesheet' type='text/css'>
+    <link rel='stylesheet' type='text/css' href='/stylesheets/normalize.css'>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/index.css">
+
+    <script type="text/javascript" src="/scripts/jquery.js" >   </script>
+    <script type="text/javascript" src="/scripts/handlebars.js"></script> 
+    <script type="text/javascript" src="/scripts/handlebarsTemplateFetcher.js"> </script>
+    <script type="text/javascript" src="/scripts/script.js" >   </script>
+    
+  </head>
+  
+  <body>
+    <div id='receipt-column'>
+      <section id='receipt-container'>
+      </section>
+    </div>
+
+    <div id='menu-column'>
+      <section id='menu-container'>
+      </section>
+    </div>
+
+    <form id='payment-form'>
+      Payment: 
```

Comment:
These forms aren't accessible, think of the blind people Ptolemy! https://developer.mozilla.org/en/docs/Web/HTML/Element